.travis.yml

```diff
@@ -42,6 +43,10 @@ matrix:
   allow_failures:
     - php: hhvm
 
+install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
```

Comment:
Sorry, I forgot to answer.

> I was afraid of a make-like behavior where you loose the cwd on each line. Is this the case ?

Every language has a way to avoid long lines (I think). For YAML, you can do it like this : https://github.com/greg0ire/enum-bundle/blob/master/.travis.yml#L19

> Which ones ?

Every one you can. Please consider that short options are for you, in your shell, but as soon as you write an option in a script or in documentation, you must be as noob-compliant (meaning explicit) as you can. Maybe this is a bit extreme for `rm -rf`, that can be considered well-known, but since you now know how to avoid long line, it is not really a problem to use `--recursive --force` instead.